docs/versions.html

```diff
@@ -20,6 +20,9 @@ <h1> PyTorch Documentation </h1>
         <li class="toctree-l1">
           <a class="reference internal" href="main/">main (unstable)</a>
         </li>
+        <li class="toctree-l1">
+          <a class="reference internal" href="2.2/">v2.2.0 (release candidate)</a>
+        </li>
         <li class="toctree-l1">
           <a class="reference internal" href="2.1/">v2.1.0 (stable release)</a>
```

Comment:
Do we need to include v2.1.1 and 2.1.2 here as well ?

Reply:
We typically do not have these links.